sig-service-catalog/charter.md

```diff
@@ -0,0 +1,131 @@
+# SIG Service Catalog Charter
+Service Catalog is a Kubernetes extension project that implements the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI).
+It allows application developers the ability to provision and consume cloud services natively from within Kubernetes.
+
+This charter adheres to the conventions described in the [Kubernetes Charter README](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md).
+
+## Scope
+
+### In scope
+
+This SIG’s main goals are:
+- Support, and adhere to, the Platform requirements of the OSBAPI specification.
+- Provide a UX for Kubernetes users that is consistent with both the OSB API specification and traditional Kubernetes user interactions.
+- Align with the [OSBAPI specification](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md) as changes are made.
+- Provide feedback (bugs or feature requests) to the OSBAPI WG.
+
+### Out of scope
+
+The following, non-exhaustive, items are out of scope:
+- Operation of OSBAPI Service Brokers.
+
+## Roles
+
+- Maintainers
+  - Responsible for approving, and reviewing, pull requests.
+  - Responsible for technical planning and stewardship of the project.
+  - New maintainers may be nominated by a chair, to accepted via lazy two-thirds
+    resolution amongst the chairs.
+  - Maintainers may be nominated for removal from their position by a chair,
+    to accepted via lazy two-thirds resolution amongst the chairs.
+
+- Chairs
```

Comment:
Chairs are listed in the sig definition https://github.com/kubernetes/community/tree/master/sig-service-catalog#chairs